renovate.json

```
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:base"
+    ],
+    "ignorePaths": [
+        "**/ui/**"
+    ],
+    "regexManagers": [
+        {
+            "fileMatch": [
+                "build-images.sh"
+            ],
+            "matchStrings": [
+                "docker\\.io\/node:(?<currentValue>[^\\s]+)"
+            ],
+            "depNameTemplate": "node",
+            "datasourceTemplate": "docker"
+        },
+        {
+            "fileMatch": [
+                "build-images.sh"
+            ],
+            "matchStrings": [
+                "docker\\.io\/ejabberd\/ecs:(?<currentValue>[^\\s]+)"
+            ],
+            "depNameTemplate": "ejabberd",
```

Renovate is not picking the update, maybe this is the issue

```suggestion
            "depNameTemplate": "ejabberd/ecs",
```